src/app/models/form.server.model.ts

```diff
@@ -61,6 +62,33 @@ import getUserModel from './user.server.model'
 
 export const FORM_SCHEMA_ID = 'Form'
 
+// Exported for testing.
+export const FORM_PUBLIC_FIELDS = [
+  'admin',
+  'authType',
+  'endPage',
+  'esrvcId',
+  'form_fields',
+  'form_logics',
+  'hasCaptcha',
+  'publicKey',
+  'startPage',
+  'status',
+  'title',
+  '_id',
+  'responseMode',
+]
+
+const POPULATE_FORM_PARAMS = {
+  path: 'admin',
+  // Remove irrelevant keys from populated fields of form admin and agency.
+  select: '-__v -created -lastModified -updatedAt -lastAccessed',
+  populate: {
+    path: 'agency',
+    select: '-__v -created -lastModified -updatedAt',
+  },
+}
```

Comment:
could we use a whitelist approach instead of a blacklist, so that the code doesn't become out of date if new keys are added to agency or user collections? i.e. explicitly select the keys to include